Gemfile

```diff
@@ -7,6 +7,7 @@ gem 'pry-byebug'
 gem 'rake', '13.0.1'
 gem 'rspec', '~> 3.9'
 gem 'rubocop', require: false
+gem 'rubocop-packaging', require: false
```

Comment:
Though it misses a `gemlock` entry but why this has been added?
Has there been any reported issues or problems that this addition helps in resolving ?

Reply:
`Gemfile.lock` has now been updated, thanks!
It helps us in maintaining downstream (links in the commit message :)). More about these cops can be found at https://docs.rubocop.org/rubocop-packaging and this has a style guide hosted at https://packaging.rubystyle.guide.

Reply:
Ok, so `require_relative` was part of this, got it :+1: